db/seeds.rb

```diff
@@ -15,7 +15,7 @@
 
     40.times do |n|
       coach = coaches.sample
-      Fabricate(:attended_session_invitation, role: "Coach", member: coach, sessions: workshop.sample )  rescue "Coach already attended"
+      Fabricate(:attended_session_invitation, role: "Coach", member: coach, workshop: workshop.sample )  rescue "Coach already attended"
```

Comment:
When I was seeding my DB I couldn't see any coaches. A validation error was being suppressed by the rescue, so I corrected the validation error.